### ISSUE TITLE
Add a Navbar relevant to the theme, i.e: space.

### ISSUE THREAD

**Issue by Mohamed-Ayaan358** [acmpesuecc/3D-Portfolio#3]
### What do we want you to do?

Currently, the website is a single-page scrollable-based website, applying a navbar to access other pages which can have their own 3D objects would be much appreciated.

I would suggest going for this if you wish to perform the issue to include more planetary bodies.

### Possible solutions

Use an asteroid belt effect to make a navbar, where each link is a planet or another body.

#### 1. Open Access time: 
 [1 hour] if each block time has passed

#### 2. Block time: 
[1 hour], to show significant progress.

**Comment by PavvanPradeep** [comment]
can i work on this

**Comment by Mohamed-Ayaan358** [comment]
yes @PavvanPradeep You may, note that it must stick to the space theme :). Bonus points if its 3d based.

**Comment by Ganther3301** [comment]
hey can i be assigned to this

**Comment by sukruthi-s** [comment]
CAN I BE ASSIGNED TO THIS (SUKRUTHI SANAMPUDI)

**Comment by Mohamed-Ayaan358** [comment]
@sukruthi-s terribly sorry but @Ganther3301 has been assigned this issue, but fret not, if he is unable to give significant updates within an hour the issue will be assigned to you, therefore you can continue the work will also be chosen as a final character in this case.If you submit your code and it is good enough half the points MAY be assigned to you.

**Comment by siddarthpai** [comment]
hey can i work on this
Siddarth.D.Pai

**Comment by PavvanPradeep** [comment]
can i work on this i have finished the code , Pavvan Pradeep

**Comment by Mohamed-Ayaan358** [comment]
@PavvanPradeep unfortunately its taken right now

**Comment by Mohamed-Ayaan358** [comment]
@Ganther3301 I expect an update within an hour

**Comment by PavvanPradeep** [comment]
@Mohamed-Ayaan358 okay lmk if its available later on cuz I have the code ready

**Comment by Mohamed-Ayaan358** [comment]
> 

Will do

**Comment by Ganther3301** [comment]
> @Ganther3301 I expect an update within an hour

so i'm done with the basic look. i'm currently working on animating it

**Comment by Mohamed-Ayaan358** [comment]
@Ganther3301 You can continue to work on it, @PavvanPradeep Make a PR, and show me updates, I cant guarantee you all the marks but I can see what I can do.

**Comment by PavvanPradeep** [comment]
@Mohamed-Ayaan358 cool will do

**Comment by Ganther3301** [comment]
hey @Mohamed-Ayaan358 i've created a pull request. could you please check it out

**Comment by siddarthpai** [comment]
hey can i work on it???? Ive made it

**Comment by Vaarun-Kamath** [comment]
hey can I work on this pls :)
